cmds/core/wget/wget_test.go

```diff
@@ -98,29 +99,36 @@ var tests = []struct {
 	},
 }
 
-func getFreePort(t *testing.T) int {
+func getListener(t *testing.T) (net.Listener, int) {
```

Comment:
I'd at least consider making the return type a TCPListener, your call. But this is pretty explicitly a function about TCP.

Reply:
Thanks for the comment. I think using Listener looks cleaner in turns of test codes. I don't need an extra net.TCPAddr type variable for the simple mocking purpose.